.github/workflows/cancel_runs.yml

```diff
@@ -0,0 +1,17 @@
+name: Cancel redundant builds
+
+on:
+  # pull_request_target runs in the context of the repository,
+  # and does *not* include any code present in the merge request.
+  # We can perform privileged actions like creating comments and
+  # cancelling runs on pull request events.
+  pull_request_target:
+
+jobs:
+  cancel-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: styfle/cancel-workflow-action@0.7.0
```

Comment:
I wonder if we should require the full commit ref or vendor the actions that actually give access to the repository. I trust github actions to not do anything dangerous, but 3rd party actions are a different story imo :)

Reply:
Yeah this is a valid concern! I'd vote for pinning to a commit hash rather than vendoring to be honest.

Reply:
I've pinned it to the commit hash for v0.8.0